Tests/ASDisplayNodeSnapshotTests.mm

```diff
@@ -61,18 +61,50 @@ - (void)testPrecompositedCornerRounding
 
 - (void)testClippingCornerRounding
 {
+  self.recordMode = YES;
+#if AS_AT_LEAST_IOS13
```

Comment:
Looking for feedback on this inline approach. Should I delete the reference images for older Xcode/iOS versions?

Reply:
I personally think this is fine for now.